src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -814,6 +815,33 @@ BOOST_AUTO_TEST_CASE(HardShrinkFunctionTest)
 }
 
 /**
+ * Basic test of the EliSH function.
+ */
+BOOST_AUTO_TEST_CASE(ElishFunctionTest)
+{
+  // Manually-calculated using python-numpy module.
+  const arma::colvec desiredActivations("-0.10307056080762242 \
+                                          3.0746696870503545 \
+                                          4.450558758162331 \
+                                          -3.0457406053548387e-44 \
+                                          0.7310585786300049 \
+                                          -0.17000340156854793 \
+                                          1.7615941559557646 0.0 ");
+
+  const arma::colvec desiredDerivatives("0.4033889157743519 \
+                                         1.0856292040894129 \
+                                         1.0392179837023456 \
+                                         0.5 0.835403899885462 \
+                                         0.3472572607652895 \
+                                         1.0737880406234939 0.5");
```

Comment:
This is a picky style comment. Since we only want to check accuracy of function (to 1% deviation) we can reduce some decimals from here and add multiple numbers in one line to make it clean and short. Let me know what you think. Thanks a lot.

Reply:
Hmm, that sounds good. Nice👍